modules/simple-sa/outputs.tf

```diff
@@ -19,10 +19,21 @@ output "email" {
   value       = google_service_account.sa.email
 }
 
+output "iam_email" {
+  description = "IAM format service account email"
+  value       = "serviceAccount:${google_service_account.sa.email}"
+}
+
 output "id" {
   description = "Service account id and email"
   value = {
     id    = google_service_account.sa.account_id,
     email = google_service_account.sa.email
   }
 }
+
+output "env_vars" {
```

Comment:
Where do we plan to use these as env vars?

Reply:
JSS solution three-tier-web-app and Gen-AI-RAG-with-cloud-sql both requires the SERVICE_ACCOUNT_EMAIL environment variable in backend service (cloud-run).
1. https://github.com/q2w/terraform-genai-rag/blob/using-adc-format/infra/main.tf#L80
2. https://github.com/q2w/solution-builder-three-tier-web-app/blob/apphub-prototype/infra/main.tf#L83

Adding this env_vars output to use it directly as env_vars in backend (cloud-run)

Reply:
gotcha, sg